Index: ms-calc/src/main/resources/application.properties
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>spring.application.name=ms-dossier\r\n\r\nserver.port=8084\r\n\r\nspring.mail.host=smtp.gmail.com\r\nspring.mail.port=25\r\nspring.mail.username=sergeevbass@gmail.com\r\nspring.mail.password=fxtr uhja qila xugp\r\nspring.mail.properties.mail.smtp.auth=true\r\nspring.mail.properties.mail.smtp.starttls.enable=true\r\n\r\nms-deal.address=localhost:8082
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>ISO-8859-1
===================================================================
diff --git a/ms-calc/src/main/resources/application.properties b/ms-calc/src/main/resources/application.properties
--- a/ms-calc/src/main/resources/application.properties	(revision ae82d932b15d09344caf6e7a2eb71e451cf962e5)
+++ b/ms-calc/src/main/resources/application.properties	(date 1720450340690)
@@ -1,12 +1,12 @@
-spring.application.name=ms-dossier
+spring.application.name=ms-calc
+server.port=8080
 
-server.port=8084
+logging.level.root=info
 
-spring.mail.host=smtp.gmail.com
-spring.mail.port=25
-spring.mail.username=sergeevbass@gmail.com
-spring.mail.password=fxtr uhja qila xugp
-spring.mail.properties.mail.smtp.auth=true
-spring.mail.properties.mail.smtp.starttls.enable=true
+springdoc.swagger-ui.path=/swagger.html
 
-ms-deal.address=localhost:8082
\ No newline at end of file
+app.main-rate=15.00
+app.base-insurance=3000
+
+app.min-age=20
+app.max-age=65
Index: ms-calc/src/main/java/ru/alex/mscalc/dto/CreditDto.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package ru.alex.mscalc.dto;\r\n\r\nimport lombok.AllArgsConstructor;\r\nimport lombok.Builder;\r\nimport lombok.Data;\r\n\r\nimport java.math.BigDecimal;\r\nimport java.util.List;\r\n\r\n@Data\r\n@AllArgsConstructor\r\n@Builder\r\n@NoArgsConstructor\r\npublic class CreditDto {\r\n\r\n    private BigDecimal amount;\r\n    private Integer term;\r\n    private BigDecimal monthlyPayment;\r\n    private BigDecimal rate;\r\n    private BigDecimal psk;\r\n    private Boolean isInsuranceEnabled;\r\n    private Boolean isSalaryClient;\r\n    private List<PaymentScheduleElementDto> paymentSchedule;\r\n}\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/ms-calc/src/main/java/ru/alex/mscalc/dto/CreditDto.java b/ms-calc/src/main/java/ru/alex/mscalc/dto/CreditDto.java
--- a/ms-calc/src/main/java/ru/alex/mscalc/dto/CreditDto.java	(revision ae82d932b15d09344caf6e7a2eb71e451cf962e5)
+++ b/ms-calc/src/main/java/ru/alex/mscalc/dto/CreditDto.java	(date 1720450340689)
@@ -3,6 +3,7 @@
 import lombok.AllArgsConstructor;
 import lombok.Builder;
 import lombok.Data;
+import lombok.NoArgsConstructor;
 
 import java.math.BigDecimal;
 import java.util.List;
Index: ms-calc/src/main/java/ru/alex/msdossier/config/KafkaConsumerConfig.java
===================================================================
diff --git a/ms-calc/src/main/java/ru/alex/msdossier/config/KafkaConsumerConfig.java b/ms-calc/src/main/java/ru/alex/msdossier/config/KafkaConsumerConfig.java
deleted file mode 100644
--- a/ms-calc/src/main/java/ru/alex/msdossier/config/KafkaConsumerConfig.java	(revision ae82d932b15d09344caf6e7a2eb71e451cf962e5)
+++ /dev/null	(revision ae82d932b15d09344caf6e7a2eb71e451cf962e5)
@@ -1,46 +0,0 @@
-package ru.alex.msdossier.config;
-
-import java.util.HashMap;
-import org.apache.kafka.clients.consumer.ConsumerConfig;
-import org.apache.kafka.common.serialization.StringDeserializer;
-import org.springframework.context.annotation.Bean;
-import org.springframework.context.annotation.Configuration;
-import org.springframework.kafka.config.ConcurrentKafkaListenerContainerFactory;
-import org.springframework.kafka.core.ConsumerFactory;
-import org.springframework.kafka.core.DefaultKafkaConsumerFactory;
-import org.springframework.kafka.support.serializer.JsonDeserializer;
-import ru.alex.msdossier.dto.EmailMessage;
-
-
-@Configuration
-public class KafkaConsumerConfig {
-
-    @Bean
-    public ConsumerFactory<String, EmailMessage> consumerFactory() {
-        var consumerProperties = new HashMap<String, Object>();
-
-        consumerProperties.put(ConsumerConfig.BOOTSTRAP_SERVERS_CONFIG, "localhost:9092");
-        consumerProperties.put(ConsumerConfig.KEY_DESERIALIZER_CLASS_CONFIG, StringDeserializer.class);
-        consumerProperties.put(ConsumerConfig.VALUE_DESERIALIZER_CLASS_CONFIG, JsonDeserializer.class);
-        consumerProperties.put(ConsumerConfig.GROUP_ID_CONFIG, "email-sender");
-        consumerProperties.put(ConsumerConfig.AUTO_OFFSET_RESET_CONFIG, "earliest");
-        consumerProperties.put(ConsumerConfig.ENABLE_AUTO_COMMIT_CONFIG, false);
-        consumerProperties.put(JsonDeserializer.USE_TYPE_INFO_HEADERS, false);
-
-        var emailMessageJsonDeserializer = new JsonDeserializer<>(EmailMessage.class);
-
-        return new DefaultKafkaConsumerFactory<>(
-            consumerProperties,
-            new StringDeserializer(),
-            emailMessageJsonDeserializer
-        );
-    }
-
-    @Bean
-    public ConcurrentKafkaListenerContainerFactory<String, EmailMessage> kafkaListenerContainerFactory() {
-        var factory = new ConcurrentKafkaListenerContainerFactory<String, EmailMessage>();
-        factory.setConsumerFactory(consumerFactory());
-
-        return factory;
-    }
-}
Index: ms-calc/src/main/java/ru/alex/msdeal/config/EmailSender.java
===================================================================
diff --git a/ms-calc/src/main/java/ru/alex/msdeal/config/EmailSender.java b/ms-calc/src/main/java/ru/alex/msdeal/config/EmailSender.java
deleted file mode 100644
--- a/ms-calc/src/main/java/ru/alex/msdeal/config/EmailSender.java	(revision ae82d932b15d09344caf6e7a2eb71e451cf962e5)
+++ /dev/null	(revision ae82d932b15d09344caf6e7a2eb71e451cf962e5)
@@ -1,21 +0,0 @@
-package ru.alex.msdeal.config;
-
-import lombok.RequiredArgsConstructor;
-import lombok.extern.slf4j.Slf4j;
-import org.springframework.kafka.core.KafkaTemplate;
-import org.springframework.stereotype.Component;
-import ru.alex.msdeal.dto.EmailMessage;
-
-
-@Slf4j
-@Component
-@RequiredArgsConstructor
-public class EmailSender {
-
-    private final KafkaTemplate<String, EmailMessage> kafkaTemplate;
-
-    public void sendMail(EmailMessage message) {
-        kafkaTemplate.send(message.theme().name(), message);
-        log.info("Message {} was send to topic {}", message, message.theme().name());
-    }
-}
Index: ms-calc/src/main/java/ru/alex/msdeal/config/KafkaProducerConfig.java
===================================================================
diff --git a/ms-calc/src/main/java/ru/alex/msdeal/config/KafkaProducerConfig.java b/ms-calc/src/main/java/ru/alex/msdeal/config/KafkaProducerConfig.java
deleted file mode 100644
--- a/ms-calc/src/main/java/ru/alex/msdeal/config/KafkaProducerConfig.java	(revision ae82d932b15d09344caf6e7a2eb71e451cf962e5)
+++ /dev/null	(revision ae82d932b15d09344caf6e7a2eb71e451cf962e5)
@@ -1,49 +0,0 @@
-package ru.alex.msdeal.config;
-
-import java.util.ArrayList;
-import java.util.HashMap;
-import org.apache.kafka.clients.admin.NewTopic;
-import org.apache.kafka.clients.producer.ProducerConfig;
-import org.apache.kafka.common.serialization.StringSerializer;
-import org.springframework.context.annotation.Bean;
-import org.springframework.context.annotation.Configuration;
-import org.springframework.kafka.core.DefaultKafkaProducerFactory;
-import org.springframework.kafka.core.KafkaAdmin;
-import org.springframework.kafka.core.KafkaTemplate;
-import org.springframework.kafka.core.ProducerFactory;
-import org.springframework.kafka.support.serializer.JsonSerializer;
-import ru.alex.msdeal.dto.EmailMessage;
-import ru.alex.msdeal.entity.constant.Theme;
-
-
-@Configuration
-public class KafkaProducerConfig {
-
-    @Bean
-    public ProducerFactory<String, EmailMessage> producerFactory() {
-        var kafkaProperties = new HashMap<String, Object>();
-
-        kafkaProperties.put(ProducerConfig.BOOTSTRAP_SERVERS_CONFIG, "localhost:9092");
-        kafkaProperties.put(ProducerConfig.KEY_SERIALIZER_CLASS_CONFIG, StringSerializer.class);
-        kafkaProperties.put(ProducerConfig.VALUE_SERIALIZER_CLASS_CONFIG, JsonSerializer.class);
-
-        return new DefaultKafkaProducerFactory<>(kafkaProperties);
-    }
-
-    @Bean
-    public KafkaTemplate<String, EmailMessage> kafkaTemplate() {
-        return new KafkaTemplate<>(producerFactory());
-    }
-
-    @Bean
-    public KafkaAdmin.NewTopics createKafkaTopics() {
-        var newTopics = new ArrayList<NewTopic>();
-
-        for (Theme theme : Theme.values()) {
-            newTopics.add(new NewTopic(theme.name(), 5, (short) 3));
-        }
-
-        return new KafkaAdmin.NewTopics(newTopics.toArray(NewTopic[]::new));
-    }
-}
-
Index: ms-calc/src/main/java/ru/alex/msdeal/dto/EmailMessage.java
===================================================================
diff --git a/ms-calc/src/main/java/ru/alex/msdeal/dto/EmailMessage.java b/ms-calc/src/main/java/ru/alex/msdeal/dto/EmailMessage.java
deleted file mode 100644
--- a/ms-calc/src/main/java/ru/alex/msdeal/dto/EmailMessage.java	(revision ae82d932b15d09344caf6e7a2eb71e451cf962e5)
+++ /dev/null	(revision ae82d932b15d09344caf6e7a2eb71e451cf962e5)
@@ -1,10 +0,0 @@
-package ru.alex.msdeal.dto;
-
-import java.util.UUID;
-import lombok.Builder;
-import ru.alex.msdeal.entity.constant.Theme;
-
-
-@Builder
-public record EmailMessage(String address, Theme theme, UUID statementId, Integer sesCode) {
-}
Index: ms-calc/src/main/java/ru/alex/msdeal/exception/SesCodeNotEqualsException.java
===================================================================
diff --git a/ms-calc/src/main/java/ru/alex/msdeal/exception/SesCodeNotEqualsException.java b/ms-calc/src/main/java/ru/alex/msdeal/exception/SesCodeNotEqualsException.java
deleted file mode 100644
--- a/ms-calc/src/main/java/ru/alex/msdeal/exception/SesCodeNotEqualsException.java	(revision ae82d932b15d09344caf6e7a2eb71e451cf962e5)
+++ /dev/null	(revision ae82d932b15d09344caf6e7a2eb71e451cf962e5)
@@ -1,7 +0,0 @@
-package ru.alex.msdeal.exception;
-
-public class SesCodeNotEqualsException extends RuntimeException {
-    public SesCodeNotEqualsException(String message) {
-        super(message);
-    }
-}
Index: ms-calc/src/main/java/ru/alex/msdossier/config/MailConfig.java
===================================================================
diff --git a/ms-calc/src/main/java/ru/alex/msdossier/config/MailConfig.java b/ms-calc/src/main/java/ru/alex/msdossier/config/MailConfig.java
deleted file mode 100644
--- a/ms-calc/src/main/java/ru/alex/msdossier/config/MailConfig.java	(revision ae82d932b15d09344caf6e7a2eb71e451cf962e5)
+++ /dev/null	(revision ae82d932b15d09344caf6e7a2eb71e451cf962e5)
@@ -1,26 +0,0 @@
-package ru.alex.msdossier.config;
-
-import org.springframework.context.annotation.Configuration;
-
-
-@Configuration
-public class MailConfig {
-
-//    @Bean
-//    public JavaMailSender getJavaMailSender() {
-//        JavaMailSenderImpl mailSender = new JavaMailSenderImpl();
-//        mailSender.setHost("smtp.gmail.com");
-//        mailSender.setPort(587);
-//
-//        mailSender.setUsername("my.gmail@gmail.com");
-//        mailSender.setPassword("password");
-//
-//        Properties props = mailSender.getJavaMailProperties();
-//        props.put("mail.transport.protocol", "smtp");
-//        props.put("mail.smtp.auth", "true");
-//        props.put("mail.smtp.starttls.enable", "true");
-//        props.put("mail.debug", "true");
-//
-//        return mailSender;
-//    }
-}
Index: ms-calc/src/main/java/ru/alex/msdossier/service/EmailKafkaListener.java
===================================================================
diff --git a/ms-calc/src/main/java/ru/alex/msdossier/service/EmailKafkaListener.java b/ms-calc/src/main/java/ru/alex/msdossier/service/EmailKafkaListener.java
deleted file mode 100644
--- a/ms-calc/src/main/java/ru/alex/msdossier/service/EmailKafkaListener.java	(revision ae82d932b15d09344caf6e7a2eb71e451cf962e5)
+++ /dev/null	(revision ae82d932b15d09344caf6e7a2eb71e451cf962e5)
@@ -1,116 +0,0 @@
-package ru.alex.msdossier.service;
-
-import java.io.IOException;
-import java.nio.file.Files;
-import java.nio.file.Path;
-import jakarta.mail.MessagingException;
-import lombok.RequiredArgsConstructor;
-import lombok.extern.log4j.Log4j2;
-import org.springframework.beans.factory.annotation.Value;
-import org.springframework.core.io.ByteArrayResource;
-import org.springframework.kafka.annotation.KafkaListener;
-import org.springframework.mail.SimpleMailMessage;
-import org.springframework.mail.javamail.JavaMailSender;
-import org.springframework.mail.javamail.MimeMessageHelper;
-import org.springframework.scheduling.annotation.Async;
-import org.springframework.stereotype.Service;
-import ru.alex.msdossier.dto.EmailMessage;
-import ru.alex.msdossier.dto.constant.Theme;
-
-
-@Log4j2
-@Service
-@RequiredArgsConstructor
-public class EmailKafkaListener {
-
-    private final JavaMailSender mailSender;
-
-    @Value("${ms-deal.address}")
-    private String msDealAddress;
-
-    @Async
-    @KafkaListener(topics = { "FINISH_REGISTRATION" }, groupId = "email-sender")
-    public void sendEmailFinishRegistration(EmailMessage message) {
-        var mailMessage = new SimpleMailMessage();
-        mailMessage.setTo(message.address());
-        mailMessage.setSubject(Theme.FINISH_REGISTRATION.name());
-        mailMessage.setText("Ваша заявка предварительно одобрена, завершите оформление");
-
-        mailSender.send(mailMessage);
-
-        log.info("Message with theme: {}, was sent to: {}", message.theme(), message.address());
-    }
-
-    @Async
-    @KafkaListener(topics = "CREATE_DOCUMENT", groupId = "email-sender")
-    public void sendEmailCreateDocument(EmailMessage message) {
-        var mailMessage = new SimpleMailMessage();
-        mailMessage.setTo(message.address());
-        mailMessage.setSubject(Theme.CREATE_DOCUMENT.name());
-        mailMessage.setText("<a href=\"http://%s/deal/document/%s/send\">Ссылка на формирование документов</a>"
-            .formatted(msDealAddress, message.statementId()));
-
-        mailSender.send(mailMessage);
-
-        log.info("Message with theme: {}, was sent to: {}", message.theme(), message.address());
-    }
-
-    @Async
-    @KafkaListener(topics = "SEND_DOCUMENT", groupId = "email-sender")
-    public void sendEmailSendDocument(EmailMessage message) throws MessagingException, IOException {
-        var mimeMessage = mailSender.createMimeMessage();
-        var mimeMessageHelper = new MimeMessageHelper(mimeMessage, true);
-
-        mimeMessageHelper.setTo(message.address());
-        mimeMessageHelper.setSubject(Theme.FINISH_REGISTRATION.name());
-        mimeMessageHelper.setText(("Документы для ознакомления находятся в письме." +
-                                   "\nЕсли вы согласны, пожалуйста перейдите по это ссылке: " +
-                                   "<a href=\"http://%s/deal/document/%s/sign\">подписать документы</a>").formatted(msDealAddress, message.statementId()));
-        mimeMessageHelper.addAttachment("agreement.txt",
-            new ByteArrayResource(Files.readAllBytes(Path.of("ms-dossier/src/main/resources/agreement.txt"))));
-
-        mailSender.send(mimeMessage);
-
-        log.info("Message with theme: {}, was sent to: {}", message.theme(), message.address());
-    }
-
-    @Async
-    @KafkaListener(topics = "SEND_SES", groupId = "email-sender")
-    public void sendEmailSesCode(EmailMessage message) {
-        var mailMessage = new SimpleMailMessage();
-        mailMessage.setTo(message.address());
-        mailMessage.setSubject(Theme.SEND_SES.name());
-        mailMessage.setText(("Для подтверждения подписания документов отправьте этот код: %d\n" +
-                             "<a href=\"http://%s/deal/document/%s/code\">На этот адрес</a>").formatted(message.sesCode(), msDealAddress, message.statementId()));
-
-        mailSender.send(mailMessage);
-
-        log.info("Message with theme: {}, was sent to: {}", message.theme(), message.address());
-    }
-
-    @Async
-    @KafkaListener(topics = "CREDIT_ISSUED", groupId = "email-sender")
-    public void sendEmailCreditIssued(EmailMessage message) {
-        var mailMessage = new SimpleMailMessage();
-        mailMessage.setTo(message.address());
-        mailMessage.setSubject(Theme.STATEMENT_DENIED.name());
-        mailMessage.setText("Поздравляю вы взяли вредит!!!");
-
-        mailSender.send(mailMessage);
-
-        log.info("Message with theme: {}, was sent to: {}", message.theme(), message.address());
-    }
-
-    @Async
-    @KafkaListener(topics = "STATEMENT_DENIED", groupId = "email-sender")
-    public void sendEmailStatementDenied(EmailMessage message) {
-        var mailMessage = new SimpleMailMessage();
-        mailMessage.setTo(message.address());
-        mailMessage.setSubject(Theme.STATEMENT_DENIED.name());
-        mailMessage.setText("Ваша заявка отклонена");
-
-        mailSender.send(mailMessage);
-
-        log.info("Message with theme: {}, was sent to: {}", message.theme(), message.address());
-    }
-}
Index: ms-calc/src/main/java/ru/alex/msdeal/entity/constant/Theme.java
===================================================================
diff --git a/ms-calc/src/main/java/ru/alex/msdeal/entity/constant/Theme.java b/ms-calc/src/main/java/ru/alex/msdeal/entity/constant/Theme.java
deleted file mode 100644
--- a/ms-calc/src/main/java/ru/alex/msdeal/entity/constant/Theme.java	(revision ae82d932b15d09344caf6e7a2eb71e451cf962e5)
+++ /dev/null	(revision ae82d932b15d09344caf6e7a2eb71e451cf962e5)
@@ -1,12 +0,0 @@
-package ru.alex.msdeal.entity.constant;
-
-public enum Theme {
-    FINISH_REGISTRATION("Окончание регистрации"),
-    CREATE_DOCUMENT("Создание документа"),
-    SEND_DOCUMENT("Документ отправлен"),
-    SEND_SES("Ses-код отправлен"),
-    CREDIT_ISSUED("Кредит выдан"),
-    STATEMENT_DENIED("Заявка отклонена");
-
-    Theme(String description){}
-}
Index: ms-calc/src/main/java/ru/alex/msdeal/exception/ClientDeniedException.java
===================================================================
diff --git a/ms-calc/src/main/java/ru/alex/msdeal/exception/ClientDeniedException.java b/ms-calc/src/main/java/ru/alex/msdeal/exception/ClientDeniedException.java
deleted file mode 100644
--- a/ms-calc/src/main/java/ru/alex/msdeal/exception/ClientDeniedException.java	(revision ae82d932b15d09344caf6e7a2eb71e451cf962e5)
+++ /dev/null	(revision ae82d932b15d09344caf6e7a2eb71e451cf962e5)
@@ -1,7 +0,0 @@
-package ru.alex.msdeal.exception;
-
-public class ClientDeniedException extends RuntimeException {
-    public ClientDeniedException(String msg) {
-        super(msg);
-    }
-}
Index: ms-calc/src/main/java/ru/alex/msdossier/MsDossierApplication.java
===================================================================
diff --git a/ms-calc/src/main/java/ru/alex/msdossier/MsDossierApplication.java b/ms-calc/src/main/java/ru/alex/msdossier/MsDossierApplication.java
deleted file mode 100644
--- a/ms-calc/src/main/java/ru/alex/msdossier/MsDossierApplication.java	(revision ae82d932b15d09344caf6e7a2eb71e451cf962e5)
+++ /dev/null	(revision ae82d932b15d09344caf6e7a2eb71e451cf962e5)
@@ -1,16 +0,0 @@
-package ru.alex.msdossier;
-
-import org.springframework.boot.SpringApplication;
-import org.springframework.boot.autoconfigure.SpringBootApplication;
-import org.springframework.scheduling.annotation.EnableAsync;
-
-
-@EnableAsync
-@SpringBootApplication
-public class MsDossierApplication {
-
-	public static void main(String[] args) {
-		SpringApplication.run(MsDossierApplication.class, args);
-	}
-
-}
Index: ms-calc/src/main/java/ru/alex/msdossier/dto/EmailMessage.java
===================================================================
diff --git a/ms-calc/src/main/java/ru/alex/msdossier/dto/EmailMessage.java b/ms-calc/src/main/java/ru/alex/msdossier/dto/EmailMessage.java
deleted file mode 100644
--- a/ms-calc/src/main/java/ru/alex/msdossier/dto/EmailMessage.java	(revision ae82d932b15d09344caf6e7a2eb71e451cf962e5)
+++ /dev/null	(revision ae82d932b15d09344caf6e7a2eb71e451cf962e5)
@@ -1,8 +0,0 @@
-package ru.alex.msdossier.dto;
-
-import java.util.UUID;
-import ru.alex.msdossier.dto.constant.Theme;
-
-
-public record EmailMessage(String address, Theme theme, UUID statementId, Integer sesCode) {
-}
Index: ms-calc/src/main/java/ru/alex/msdeal/dto/SesCodeDto.java
===================================================================
diff --git a/ms-calc/src/main/java/ru/alex/msdeal/dto/SesCodeDto.java b/ms-calc/src/main/java/ru/alex/msdeal/dto/SesCodeDto.java
deleted file mode 100644
--- a/ms-calc/src/main/java/ru/alex/msdeal/dto/SesCodeDto.java	(revision ae82d932b15d09344caf6e7a2eb71e451cf962e5)
+++ /dev/null	(revision ae82d932b15d09344caf6e7a2eb71e451cf962e5)
@@ -1,13 +0,0 @@
-package ru.alex.msdeal.dto;
-
-import javax.validation.constraints.NotNull;
-import lombok.Getter;
-import lombok.Setter;
-
-
-@Setter
-@Getter
-public class SesCodeDto {
-    @NotNull
-    private int sesCode;
-}
Index: ms-calc/src/main/java/ru/alex/msdossier/dto/constant/Theme.java
===================================================================
diff --git a/ms-calc/src/main/java/ru/alex/msdossier/dto/constant/Theme.java b/ms-calc/src/main/java/ru/alex/msdossier/dto/constant/Theme.java
deleted file mode 100644
--- a/ms-calc/src/main/java/ru/alex/msdossier/dto/constant/Theme.java	(revision ae82d932b15d09344caf6e7a2eb71e451cf962e5)
+++ /dev/null	(revision ae82d932b15d09344caf6e7a2eb71e451cf962e5)
@@ -1,19 +0,0 @@
-package ru.alex.msdossier.dto.constant;
-
-import lombok.Getter;
-
-
-@Getter
-public enum Theme {
-    FINISH_REGISTRATION("Окончание регистрации"),
-    CREATE_DOCUMENT("Создание документа"),
-    SEND_DOCUMENT("Документ отправлен"),
-    SEND_SES("Ses-код отправлен"),
-    CREDIT_ISSUED("Кредит выдан"),
-    STATEMENT_DENIED("Заявка отклонена");
-
-    private String description;
-
-    Theme(String description) {
-    }
-}
